### PR TITLE
Updating version information

### PR DESCRIPTION
**Pull request by mortonjt**

### CODE REVIEW

**Comment by wasade** [comment]
flake8

**Comment by mortonjt** [comment]
@wasade merge?

File: scripts/deblur [hunk 1]
```diff
@@ -31,8 +31,7 @@ from deblur.workflow import (launch_workflow, trim_seqs, dereplicate_seqs,
 
 from deblur.support_files import pos_db, neg_db
 
-__version__ = "0.9.2.1"
-
+__version__ = "0.1.5"
```
Comment (mortonjt) [inline scripts/deblur]: Fixed!

**Comment by mortonjt** [comment]
All comments have been addressed, and its on pypi.
https://pypi.python.org/pypi?:action=pkg_edit&name=deblur

So this should be good to go

**Comment by mortonjt** [comment]
OK.  Found an aliasing problem - so I'm bumping up the version number and going to resubmit pypi package.  Please review this carefully and merge - then I can reupload to pypi.

**Comment by wasade** [comment]
Will wait for tests but :+1:

**Comment by mortonjt** [comment]
Cool.  Want to merge @wasade ?

**Pull request merged**